pkgs/applications/audio/vcv-rack/default.nix

```diff
@@ -0,0 +1,114 @@
+{ stdenv, makeWrapper, fetchFromBitbucket, fetchFromGitHub, pkgconfig
+, alsaLib, curl, glew, glfw, gtk2-x11, jansson, libjack2, libXext, libXi
+, libzip, rtaudio, rtmidi, speex }:
+
+let
+  glfw-git = glfw.overrideAttrs (oldAttrs: {
+    name = "glfw-git-20180529";
+    src = fetchFromGitHub {
+      owner = "glfw";
+      repo = "glfw";
+      rev = "0be4f3f75aebd9d24583ee86590a38e741db0904";
+      sha256 = "0zbcjgc7ks25yi949k0wjknfl00a4dqmz45mhp00k62vlq2sj0i5";
+    };
+    buildInputs = oldAttrs.buildInputs ++ [ libXext libXi ];
+  });
+  nanosvg-source = fetchFromGitHub {
+    owner = "memononen";
+    repo = "nanosvg";
+    rev = "06c1f0f3bb041d69a73bb74067d063a700215b0e";
+    sha256 = "0c4fv9zwsyi9jwcd2rc1i2l3qpdx863gkc6cw3vcfqs9h7yj9l94";
+  };
+  nanovg-source = fetchFromGitHub {
+    owner = "memononen";
+    repo = "nanovg";
+    rev = "98e551351d020087b01ae0b1d3d2d1593e3ad01f";
+    sha256 = "08mr3d1grr873mxphmhan1v8bz6nbwc1yzjpwzrn86imi2l0v3sw";
+  };
+  osdialog-source = fetchFromGitHub {
+    owner = "AndrewBelt";
+    repo = "osdialog";
+    rev = "f2624f7cee98aa51b80ffbd9efd47495638890da";
+    sha256 = "0arm3w0kfrljidxyigqdx18hj0bdrw9a5krdqy9ii0dpv580ky1k";
+  };
+  oui-blendish-source = fetchFromGitHub {
+    owner = "AndrewBelt";
+    repo = "oui-blendish";
+    rev = "b7066201022a757cbcbd986d8c91d565e4daef90";
+    sha256 = "1jbhzffz8qclkhhdzhm8fikkjcx8ksccj08lhjwcxcwiqb7wly8z";
+  };
+  pfft-source = fetchFromBitbucket {
+    owner = "jpommier";
+    repo = "pffft";
+    rev = "29e4f76ac53bef048938754f32231d7836401f79";
+    sha256 = "084csgqa6f1a270bhybjayrh3mpyi2jimc87qkdgsqcp8ycsx1l1";
+  };
+in
+with stdenv.lib; stdenv.mkDerivation rec {
+  name = "VCV-Rack-${version}";
+  version = "0.6.2b";
+
+  src = fetchFromGitHub {
+    owner = "VCVRack";
+    repo = "Rack";
+    rev = "v${version}";
+    sha256 = "0mn33aiafvqvds4hf9hywsr6f6y8y25g9vnjksf3cavxvnszy47y";
+  };
```

Comment:
You can just add `fetchSubodules = true;` here instead of fetching each dependency manually. Will also make it work for future updates. (Note that the hash will have to change).

Reply:
I wasn't sure which way was better. I changed it back.